app/assets/stylesheets/styles.scss

```diff
@@ -61,3 +63,13 @@ th .desc:after {
 #relic {
   color: $relic_color;
 }
+
+.find-panel {
+  background-color: $find_panel_bg;
+  padding: 10px;
```

Comment:
Properties should be ordered background-color, border-color, border-style, border-width, font-size, margin-top, padding